__tests__/applications/applications.test.js

```diff
@@ -0,0 +1,175 @@
+const puppeteer = require('puppeteer');
+
+const {
+  fetchedApplications,
+  acceptedApplications,
+} = require('../../mock-data/applications');
+const { superUserForAudiLogs } = require('../../mock-data/users');
+
+const SITE_URL = 'http://localhost:8000';
```

Comment:
is site URL will be localhost:8000, please confirm once

Reply:
for tests it it